### ISSUE TITLE
Pagination - remove input

### ISSUE THREAD

**Issue by LaurenFBaily** [wellcomecollection/wellcomecollection.org#10684]
# What?
Remove the input feature on the pagination component.

# Why?
There are related accessibility issues and several bugs, plus the user need is not evidenced. The url can be hacked to achieve the same result (page=3)

# Background
Issues highlighted in the 2023 external Hassell Inclusion audit. There are several live bugs relating to this feature:
#10575
#10355
#10470

# Done when
-  Users cannot input a page number in the pagination component (next or previous are the only interactive elements)
- Above related bugs are resolved

**Comment by rcantin-w** [comment]
In prod
